project.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<p>How can wesee that from the data? And why is the spread relevant to that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-42FraMeWork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>